src/main.ts

```diff
@@ -4,13 +4,14 @@ import * as exec from '@actions/exec';
 import * as io from '@actions/io';
 
 import { FormatType, SecretParser } from 'actions-secret-parser';
+import { ServicePrincipalLogin } from './PowerShell/ServicePrincipalLogin';
 
 var azPath: string;
 var prefix = !!process.env.AZURE_HTTP_USER_AGENT ? `${process.env.AZURE_HTTP_USER_AGENT}` : "";
 
 async function main() {
-    try{
-        // Set user agent varable
+    try {
+        // Set user agent variable

```

Comment:
Can we add comments like:
Attempting Az cli login
Attempting Az PS login
Login via Az cli is successful.
Login via Az pS is successful